health_imaging.py

```diff
@@ -22,6 +22,14 @@ class ImagingTestRequest(metaclass=PoolMeta):
 
     study_instance_uid = fields.Char('Study Instance UID')
 
+    ACCESSION_NUMBER_PREFIX = config.get(
```

Comment:
My python is bad and I'm also not sure exactly how GNUHealth is using these classes, but it seems like it must be basing some logic on reflection and the fields present.
Is there any danger with specifying `ACCESSION_NUMBER_PREFIX` within the class as a result? Could something read/set this value by accident if it's made public in the same way that `study_instance_uid` and `accession_number` are?

Reply:
It's a great question and I wasn't sure as well. I tried to find out and it seems that they only care about `class` attributes that are instances of `Field` class: https://github.com/tryton/trytond/blob/develop/trytond/model/model.py#L76